tools/busywork/tcl/fabric.tcl

```diff
@@ -357,7 +357,7 @@ proc ::fabric::caLogin {i_peer i_user i_secret} {
     http::cleanup $token
 }
 
-    
```

Comment:
@ghaskins I'm not worried about the white space. Is there an Emacs setting that does this automatically (whitespace cleanup)? I wasn't aware of one.